diff --git a/browser/components/abouttor/AboutTorParent.sys.mjs b/browser/components/abouttor/AboutTorParent.sys.mjs
index 8449722c3b03..9d6a97e61b46 100644
--- a/actors/AboutTorParent.sys.mjs
+++ b/actors/AboutTorParent.sys.mjs
@@ -10,6 +10,7 @@ ChromeUtils.defineESModuleGetters(lazy, {
 const initializedActors = new Set();
 const onionizePref = "torbrowser.homepage.search.onionize";
 const surveyDismissVersionPref = "torbrowser.homepage.survey.dismiss_version";
+const tailsPref = "torbrowser.running_in_tails";
 
 /**
  * Actor parent class for the about:tor page.
@@ -69,6 +70,7 @@ export class AboutTorParent extends JSWindowActorParent {
       ),
       appLocale,
       dismissYEC: AboutTorParent.#dismissYEC,
+      isTails: Services.prefs.getBoolPref(tailsPref, false),
     };
   }
 
diff --git a/browser/components/abouttor/content/aboutTor.css b/browser/components/abouttor/content/aboutTor.css
index 5140aac1aa78..e02685765650 100644
--- a/chrome/browser/content/browser/abouttor/aboutTor.css
+++ b/chrome/browser/content/browser/abouttor/aboutTor.css
@@ -13,7 +13,9 @@ body {
     "tor-check tor-check tor-check" auto
     ". form ." min-content
     "message message message" auto
+    "tails tails tails" auto
     "survey survey survey" auto
+    "check check check" auto
     /* End space: unfilled.
      * Reserve 150px for background image.
      * NOTE: Since the body has "auto" height, the other "1fr" flex row will
@@ -525,3 +527,36 @@ body.show-yec h1 {
     --in-content-button-background-active: var(--yec-button-background-active);
   }
 }
+
+/* Tails customization */
+
+.is-tails .home-message {
+  display: none;
+}
+
+body:not(.is-tails) #tails-message {
+  display: none;
+}
+
+#tails-message {
+  grid-area: tails;
+  font-weight: 400;
+  text-align: center;
+  margin-block: 0.6em;
+}
+
+#tails-message ul {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  padding-inline-start: 0;
+}
+
+body:not(.is-tails) #tails-tor-check {
+  display: none;
+}
+
+#tails-tor-check {
+  grid-area: check;
+  margin-block: 0 1.6em;
+}
diff --git a/browser/components/abouttor/content/aboutTor.html b/browser/components/abouttor/content/aboutTor.html
index 4e181d41d9bf..53a01cfdd38f 100644
--- a/chrome/browser/content/browser/abouttor/aboutTor.html
+++ b/chrome/browser/content/browser/abouttor/aboutTor.html
@@ -162,6 +162,46 @@
         ></a>
       </span>
     </p>
+    <div id="tails-message">
+      <p>
+        <img
+          class="message-emoji"
+          alt=""
+          src="chrome://browser/content/abouttor/2728-sparkles.svg"
+        />
+        <span data-l10n-id="tails-release-notes">
+          <a
+            data-l10n-name="tails-release-notes-link"
+            href="https://tails.net/news/version_7.4.2"
+            target="_blank"
+          ></a>
+        </span>
+      </p>
+      <p>
+        <img
+          class="message-emoji"
+          alt=""
+          src="chrome://browser/content/abouttor/1f4e3-megaphone.svg"
+        />
+        <span data-l10n-id="tails-newsletter"></span>
+      </p>
+      <ul>
+        <li>
+          <a
+            data-l10n-id="tails-newsletter-tails-link"
+            href="https://tails.net/news"
+            target="_blank"
+          ></a>
+        </li>
+        <li>
+          <a
+            data-l10n-id="tails-newsletter-tor-link"
+            href="https://newsletter.torproject.org"
+            target="_blank"
+          ></a>
+        </li>
+      </ul>
+    </div>
     <!-- Survey element, initially used for tor-browser#43504. -->
     <article id="survey" class="tor-home-box" aria-labelledby="survey-heading">
       <img
@@ -183,5 +223,12 @@
         iconSrc="chrome://global/skin/icons/close.svg"
       ></moz-button>
     </article>
+    <p id="tails-tor-check">
+      <a
+        data-l10n-id="tails-tor-check-link"
+        href="https://check.torproject.org/"
+        target="_blank"
+      ></a>
+    </p>
   </body>
 </html>
diff --git a/browser/components/abouttor/content/aboutTor.js b/browser/components/abouttor/content/aboutTor.js
index 8b718027d0b4..01c9676f2584 100644
--- a/chrome/browser/content/browser/abouttor/aboutTor.js
+++ b/chrome/browser/content/browser/abouttor/aboutTor.js
@@ -85,6 +85,7 @@ const MessageArea = {
   _initialized: false,
   _messageData: null,
   _isStable: null,
+  _isTails: null,
   _torConnectEnabled: null,
 
   /**
@@ -103,11 +104,13 @@ const MessageArea = {
    * @param {boolean} isStable - Whether this is the stable release version.
    * @param {boolean} torConnectEnabled - Whether TorConnect is enabled, and
    *   therefore the Tor process was configured with about:torconnect.
+   * @param {boolean} isTails - Whether we're running in Tails.
    */
-  setMessageData(messageData, isStable, torConnectEnabled) {
+  setMessageData(messageData, isStable, torConnectEnabled, isTails) {
     this._messageData = messageData;
     this._isStable = isStable;
     this._torConnectEnabled = torConnectEnabled;
+    this._isTails = isTails;
     this._update();
   },
 
@@ -127,7 +130,9 @@ const MessageArea = {
     // Set heading.
     document.body.classList.toggle("is-testing", !this._isStable);
 
-    document.body.classList.toggle("show-tor-check", !this._torConnectEnabled);
+    document.body.classList.toggle("show-tor-check", !this._torConnectEnabled && !this._isTails);
+
+    document.body.classList.toggle("is-tails", this._isTails);
 
     const { updateVersion, updateURL, number } = this._messageData;
 
@@ -574,9 +579,15 @@ window.addEventListener("InitialData", event => {
     surveyDismissVersion,
     appLocale,
     dismissYEC,
+    isTails,
   } = event.detail;
   SearchWidget.setOnionizeState(!!searchOnionize);
-  MessageArea.setMessageData(messageData, !!isStable, !!torConnectEnabled);
+  MessageArea.setMessageData(
+    messageData,
+    !!isStable,
+    !!torConnectEnabled,
+    !!isTails
+  );
   SurveyArea.potentiallyShow(surveyDismissVersion, isStable, appLocale);
   YecArea.potentiallyShow(dismissYEC, isStable, appLocale);
 
