diff -Naur browser-omni.orig/actors/AboutTorParent.sys.mjs browser-omni/actors/AboutTorParent.sys.mjs
--- browser-omni.orig/actors/AboutTorParent.sys.mjs	2025-11-10 10:30:00.000000000 +0100
+++ browser-omni/actors/AboutTorParent.sys.mjs	2025-11-11 15:27:50.937672061 +0100
@@ -69,6 +69,7 @@
       ),
       appLocale,
       dismissYEC: AboutTorParent.#dismissYEC,
+      isTails: Services.prefs.getBoolPref("torbrowser.running_in_tails", false),
     };
   }
 
diff -Naur browser-omni.orig/chrome/browser/content/browser/abouttor/aboutTor.css browser-omni/chrome/browser/content/browser/abouttor/aboutTor.css
--- browser-omni.orig/chrome/browser/content/browser/abouttor/aboutTor.css	2025-11-10 10:30:00.000000000 +0100
+++ browser-omni/chrome/browser/content/browser/abouttor/aboutTor.css	2025-11-11 15:26:23.108478823 +0100
@@ -13,7 +13,9 @@
     "tor-check tor-check tor-check" auto
     ". form ." min-content
     "message message message" auto
+    "tails tails tails" auto
     "survey survey survey" auto
+    "check check check" auto
     /* End space: unfilled.
      * Reserve 150px for background image.
      * NOTE: Since the body has "auto" height, the other "1fr" flex row will
@@ -532,3 +534,36 @@
     --in-content-button-background-active: var(--yec-button-background-active);
   }
 }
+
+/* Tails customization */
+
+.is-tails .home-message {
+  display: none;
+}
+
+body:not(.is-tails) #tails-message {
+  display: none;
+}
+
+#tails-message {
+  grid-area: tails;
+  font-weight: 400;
+  text-align: center;
+  margin-block: 0.6em;
+}
+
+#tails-message ul {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  padding-inline-start: 0;
+}
+
+body:not(.is-tails) #tails-tor-check {
+  display: none;
+}
+
+#tails-tor-check {
+  grid-area: check;
+  margin-block: 0 1.6em;
+}
diff -Naur browser-omni.orig/chrome/browser/content/browser/abouttor/aboutTor.html browser-omni/chrome/browser/content/browser/abouttor/aboutTor.html
--- browser-omni.orig/chrome/browser/content/browser/abouttor/aboutTor.html	2025-11-10 10:30:00.000000000 +0100
+++ browser-omni/chrome/browser/content/browser/abouttor/aboutTor.html	2025-11-11 15:26:23.108639320 +0100
@@ -170,6 +170,46 @@
         ></a>
       </span>
     </p>
+    <div id="tails-message">
+      <p>
+        <img
+          class="message-emoji"
+          alt=""
+          src="chrome://browser/content/abouttor/2728-sparkles.svg"
+        />
+        <span data-l10n-id="tails-release-notes">
+          <a
+            data-l10n-name="tails-release-notes-link"
+            href="https://tails.net/news/version_7.1"
+            target="_blank"
+          ></a>
+        </span>
+      </p>
+      <p>
+        <img
+          class="message-emoji"
+          alt=""
+          src="chrome://browser/content/abouttor/1f4e3-megaphone.svg"
+        />
+        <span data-l10n-id="tails-newsletter"></span>
+      </p>
+      <ul>
+        <li>
+          <a
+            data-l10n-id="tails-newsletter-tails-link"
+            href="https://tails.net/news"
+            target="_blank"
+          ></a>
+        </li>
+        <li>
+          <a
+            data-l10n-id="tails-newsletter-tor-link"
+            href="https://newsletter.torproject.org"
+            target="_blank"
+          ></a>
+        </li>
+      </ul>
+    </div>
     <!-- Survey element, initially used for tor-browser#43504. -->
     <article id="survey" class="tor-home-box" aria-labelledby="survey-heading">
       <img
@@ -191,5 +231,12 @@
         iconSrc="chrome://global/skin/icons/close.svg"
       ></moz-button>
     </article>
+    <p id="tails-tor-check">
+      <a
+        data-l10n-id="tails-tor-check-link"
+        href="https://check.torproject.org/"
+        target="_blank"
+      ></a>
+    </p>
   </body>
 </html>
diff -Naur browser-omni.orig/chrome/browser/content/browser/abouttor/aboutTor.js browser-omni/chrome/browser/content/browser/abouttor/aboutTor.js
--- browser-omni.orig/chrome/browser/content/browser/abouttor/aboutTor.js	2025-11-10 10:30:00.000000000 +0100
+++ browser-omni/chrome/browser/content/browser/abouttor/aboutTor.js	2025-11-11 15:26:23.108792881 +0100
@@ -85,6 +85,7 @@
   _initialized: false,
   _messageData: null,
   _isStable: null,
+  _isTails: null,
   _torConnectEnabled: null,
 
   /**
@@ -103,11 +104,13 @@
    * @param {boolean} isStable - Whether this is the stable release version.
    * @param {boolean} torConnectEnabled - Whether TorConnect is enabled, and
    *   therefore the Tor process was configured with about:torconnect.
+   * @param {boolean} isTails - Whether we're running in Tails.
    */
-  setMessageData(messageData, isStable, torConnectEnabled) {
+  setMessageData(messageData, isStable, torConnectEnabled, isTails) {
     this._messageData = messageData;
     this._isStable = isStable;
     this._torConnectEnabled = torConnectEnabled;
+    this._isTails = isTails;
     this._update();
   },
 
@@ -127,7 +130,9 @@
     // Set heading.
     document.body.classList.toggle("is-testing", !this._isStable);
 
-    document.body.classList.toggle("show-tor-check", !this._torConnectEnabled);
+    document.body.classList.toggle("show-tor-check", !this._torConnectEnabled && !this._isTails);
+
+    document.body.classList.toggle("is-tails", this._isTails);
 
     const { updateVersion, updateURL, number } = this._messageData;
 
@@ -602,9 +607,15 @@
     surveyDismissVersion,
     appLocale,
     dismissYEC,
+    isTails,
   } = event.detail;
   SearchWidget.setOnionizeState(!!searchOnionize);
-  MessageArea.setMessageData(messageData, !!isStable, !!torConnectEnabled);
+  MessageArea.setMessageData(
+    messageData,
+    !!isStable,
+    !!torConnectEnabled,
+    !!isTails
+  );
   SurveyArea.potentiallyShow(surveyDismissVersion, isStable, appLocale);
   YecArea.potentiallyShow(dismissYEC, isStable, appLocale);
 
