diff -Naur a/localization/en-US/toolkit/global/tor-browser.ftl b/localization/en-US/toolkit/global/tor-browser.ftl
--- a/localization/en-US/toolkit/global/tor-browser.ftl	2025-11-26 13:59:39.956124476 +0100
+++ b/localization/en-US/toolkit/global/tor-browser.ftl	2025-11-26 13:59:39.958693408 +0100
@@ -28,6 +28,17 @@
 tor-browser-home-message-testing = This is an unstable version of Tor Browser for testing new features. <a data-l10n-name="learn-more-link">Learn more</a>
 
 
+tails-release-notes = You are running Tails 7.2. <a data-l10n-name="tails-release-notes-link">See what’s new</a>
+
+tails-newsletter = Get the latest news from Tails and Tor straight to your inbox.
+
+tails-newsletter-tails-link = Sign up for Tails news
+
+tails-newsletter-tor-link = Sign up for Tor news
+
+tails-tor-check-link = Tor check
+
+
 home-mode-choice-tor =
     .label = Tor Browser Home
 
