diff -Naur omni.orig/localization/en-US/toolkit/global/tor-browser.ftl omni.new/localization/en-US/toolkit/global/tor-browser.ftl
--- omni.orig/localization/en-US/toolkit/global/tor-browser.ftl	2000-01-01 00:00:00.000000000 +0000
+++ omni.new/localization/en-US/toolkit/global/tor-browser.ftl	2025-10-09 10:10:42.952000000 +0000
@@ -27,6 +27,20 @@
 
 tor-browser-home-message-testing = This is an unstable version of Tor Browser for testing new features. <a data-l10n-name="learn-more-link">Learn more</a>
 
+## Tor browser home page for Tails.
+
+# "Tails" refers to the operating system, and should be translated as a brand name.
+# '<a data-l10n-name="tails-release-notes-link">' should contain the link text and be closed with '</a>'.
+tails-release-notes = You are running Tails 7.2. <a data-l10n-name="tails-release-notes-link">See what’s new</a>
+
+tails-newsletter = Get the latest news from Tails and Tor straight to your inbox.
+
+tails-newsletter-tails-link = Sign up for Tails news
+
+tails-newsletter-tor-link = Sign up for Tor news
+
+tails-tor-check-link = Tor check
+
 
 home-mode-choice-tor =
     .label = Tor Browser Home
